### ISSUE TITLE
Nan in summary histogram for: SecondStageFeatureExtractor/resnet_v1_50/block4/unit_3/bottleneck_v1/conv3/BatchNorm/gamma_1

### ISSUE THREAD

**Issue by luyuesheng** [tensorflow/models#2730]
I attemp to run   object_detection\samples\configs\faster_rcnn_resnet50_pets.config. After some global steps,my program meet a bug.

System information
•win7-64
•TensorFlow installed from binary
•TensorFlow 1.3
•Python version:3.5
•CUDA/cuDNN version: cuda8.0/cuDNN6.0
•GPU model and memory: gtx1060/6G

INFO:tensorflow:global step 186: loss = 4.5385 (1.394 sec/step)
INFO:tensorflow:global_step/sec: 1.55281
2017-11-08 08:05:39.363437: W C:\tf_jenkins\home\workspace\rel-win\M\windows-gpu\PY\35\tensorflow\core\framework\op_kernel.cc:1192] Invalid argument: Nan in summary histogram for: SecondStageFeatureExtractor/resnet_v1_50/block4/unit_3/bottleneck_v1/conv3/BatchNorm/gamma_1

Caused by op 'SecondStageFeatureExtractor/resnet_v1_50/block4/unit_3/bottleneck_v1/conv3/BatchNorm/gamma_1', defined at:
  File "D:/TF_Model/research/object_detection/train.py", line 164, in <module>
    tf.app.run()

**Comment by bignamehyp** [comment]
Please sync to t f1.4 to see if this issue still exists.

**Comment by luyuesheng** [comment]
thank you!    After running  " pip install tf-nightly-gpu  ",my program seems normal,now global step is 1007.  still is training!
